extensions/amp-social-share/1.0/amp-social-share.css

```diff
@@ -20,6 +20,16 @@
   * See https://github.com/ampproject/amphtml/issues/4277 for more details.
   */
 
+amp-social-share:focus-within {
+  outline: #0389ff solid 2px;
+  outline-offset: 2px;
```

Comment:
What's the offset for?

Reply:
The outline offset was recommended in #29561 for 0.1